_posts/2012-07-10-on-demand-jenkins-slaves-with-amazon-ec2.markdown

```diff
@@ -1,55 +1,80 @@
 ---
 layout: post
-title: "On-Demand Jenkins Slaves with Amazon EC2"
+title: "On-Demand Jenkins Agents with Amazon EC2"
 date: 2012-07-10 13:30
 comments: true
 categories: [Jenkins, Testing, Continuous Integration, EC2]
 author: [joey, frank]
 ---
 
-The [Artsy](http://artsy.net) team faithfully uses [Jenkins](http://jenkins-ci.org) for continuous integration. [As we've described before](http://artsy.github.com/blog/2012/05/27/using-jenkins-for-ruby-and-ruby-on-rails-teams/), our Jenkins master and 8 slaves run on Linode. This arrangement has at least a few drawbacks:
+The [Artsy](http://artsy.net) team faithfully uses [Jenkins](http://jenkins-ci.org) for continuous integration.
+[As we've described before](http://artsy.github.com/blog/2012/05/27/using-jenkins-for-ruby-and-ruby-on-rails-teams/),
+our Jenkins controller and 8 agents run on Linode. This arrangement has at least a few drawbacks:
```

Comment:
bummer, but there are actual changes mixed in with formatting changes here. Basically I changed every instance of master to controller, and every instance of slave to agent.

Reply:
Is it possible to break those into two separate commits? It might not be worth it retroactively, but that would probably be ideal.

Reply:
🤷 It doesn't personally bother me because we have this PR, but if you think it's important that they are separated, let me know and I'll re-do the commits.